source/drivers/null/ur_nullddi.cpp

```diff
@@ -179,37 +172,35 @@ __urdlllocal ur_result_t UR_APICALL urContextSetExtendedDeleter(
 __urdlllocal ur_result_t UR_APICALL urEnqueueKernelLaunch(
     ur_queue_handle_t hQueue,   ///< [in] handle of the queue object
     ur_kernel_handle_t hKernel, ///< [in] handle of the kernel object
-    uint32_t workDim, ///< [in] number of dimensions, from 1 to 3, to specify
-                      ///< the global and work-group work-items
-    const size_t
-        *pGlobalWorkOffset, ///< [in] pointer to an array of workDim unsigned
-                            ///< values that specify the offset used to
-                            ///< calculate the global ID of a work-item
-    const size_t *pGlobalWorkSize, ///< [in] pointer to an array of workDim
-                                   ///< unsigned values that specify the number
-                                   ///< of global work-items in workDim that
-                                   ///< will execute the kernel function
-    const size_t
-        *pLocalWorkSize, ///< [in][optional] pointer to an array of workDim
-                         ///< unsigned values that specify the number of local
-                         ///< work-items forming a work-group that will execute
-                         ///< the kernel function. If nullptr, the runtime
-                         ///< implementation will choose the work-group size.
+    uint32_t
+        workDim, ///< [in] number of dimensions, from 1 to 3, to specify the global and
+                 ///< work-group work-items
+    const size_t *
+        pGlobalWorkOffset, ///< [in] pointer to an array of workDim unsigned values that specify the
+    ///< offset used to calculate the global ID of a work-item
+    const size_t *
+        pGlobalWorkSize, ///< [in] pointer to an array of workDim unsigned values that specify the
+    ///< number of global work-items in workDim that will execute the kernel
+    ///< function
+    const size_t *
+        pLocalWorkSize, ///< [in][optional] pointer to an array of workDim unsigned values that
+    ///< specify the number of local work-items forming a work-group that will
+    ///< execute the kernel function.
+    ///< If nullptr, the runtime implementation will choose the work-group
+    ///< size.
```

Comment:
seems like the old breaks are still present but are no longer aligned, was this originally one long line before the previous formatting?